code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -513,13 +513,14 @@
 	return //Sprites required for flashlight
 
 /obj/item/clothing/suit/space/hardsuit/captain
-	name = "captain's hardsuit"
-	desc = "A bulky, heavy-duty piece of exclusive Nanotrasen hardsuit armor. YOU are in charge!"
+	name = "captain's SWAT suit"
+	desc = "A MK.II SWAT suit with streamlined joints and armor made out of superior materials, insulated against intense heat. The most advanced tactical armor available Usually reserved for heavy hitter corporate security, this one has a regal finish in Nanotrasen company colors. Better not let the assistants get a hold of it."
 	icon_state = "caparmor"
 	item_state = "capspacesuit"
 	allowed = list(/obj/item/weapon/tank/internals, /obj/item/device/flashlight,/obj/item/weapon/gun/energy, /obj/item/weapon/gun/ballistic, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs)
-	armor = list(melee = 40, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50, fire = 100, acid = 100)
+	armor = list(melee = 80, bullet = 50, laser = 70, energy = 25, bomb = 50, bio = 100, rad = 50, fire = 100, acid = 100)
```

Comment:
its always important to note that dr has made armor a lot less relevant than it was when it was rng to block all damage.
a 40% to 80% jump in probability to block all damage (and then if that fails, a reroll to block half damage (and then another reroll for 4/5ths damage)) is a lot more meaningful than a dam*.6 jumping to dam*.2